db/trie/extensionnode.go

```diff
@@ -32,27 +29,27 @@ func newExtensionNodeFromProtoPb(pb *iproto.ExtendPb) *extensionNode {
 	}
 }
 
-func (e *extensionNode) Children(ctx context.Context) ([]Node, error) {
-	tc, ok := getSameKeyLenTrieContext(ctx)
-	if !ok {
-		return nil, errors.New("failed to get operation context")
-	}
-	child, err := e.child(tc)
-	if err != nil {
-		return nil, err
-	}
-
-	return []Node{child}, nil
-}
-
 func (e *extensionNode) Type() NodeType {
 	return EXTENSION
 }
 
+func (e *extensionNode) Key() []byte {
+	return nil
+}
+
 func (e *extensionNode) Value() []byte {
 	return nil
 }
 
+func (e *extensionNode) children(tc SameKeyLenTrieContext) ([]Node, error) {
```

Comment:
If no longer being an interface, should we change it to child(tx) (Node, error)

Reply:
will address this in next PR